tests/net/lib/dns_addremove/prj.conf

```diff
@@ -9,12 +9,10 @@ CONFIG_DNS_NUM_CONCUR_QUERIES=1
 CONFIG_NET_LOG=y
 CONFIG_NET_MGMT=y
 CONFIG_NET_MGMT_EVENT=y
-CONFIG_NET_MGMT_EVENT_STACK_SIZE=768
 CONFIG_NET_MGMT_EVENT_THREAD_PRIO=7
 CONFIG_NET_MGMT_EVENT_QUEUE_SIZE=2
 CONFIG_NET_IPV4=y
 CONFIG_NET_IPV6=y
 CONFIG_NET_ARP=n
 CONFIG_PRINTK=y
 CONFIG_ZTEST=y
-CONFIG_MAIN_STACK_SIZE=1344
```

Comment:
Hmmm dns_addremove test is now failing in CI. But the main stack size shouldn't matter, it runs under ztest anyway. We also ought to be getting stack overflow exceptions. Need to look into this further.

Reply:
The dns_addremove test is failing with a 'recursive spinlock' error apparently.